templates/bridgeserver2.yaml

```diff
@@ -0,0 +1,834 @@
+# This CF script is for creating an AWS elastic beanstalk application.
+#
+# Usage:
+#   1. Run the boostrap script to setup required resources for the app.
+#   2. Create the app with `aws cloudformation --create-stack` command
+#   3. Update the stack with `aws cloudformation --update-stack` command
+#
+# initial creation:
+#   aws --profile aws.admin.user --region us-east-1 cloudformation \
+#     create-stack --stack-name foo-develop --capabilities CAPABILITY_NAMED_IAM \
+#     --on-failure DELETE --template-body file://cf_templates/eb_app.yml \
+#     --parameters ParameterKey=EC2InstanceType,ParameterValue=t2.micro \
+#     ...[all params neeeded for app]
+#
+# updates:
+#   aws --profile aws.bootstrap.user --region us-east-1 cloudformation \
+#     update-stack --stack-name foo-develop --capabilities CAPABILITY_NAMED_IAM \
+#     --template-body file://cf_templates/eb_app.yml \
+#     --parameters ParameterKey=EC2InstanceType,ParameterValue=t2.micro \
+#     ...[all params neeeded for app]
+#
+Description: Resources for BridgeServer2 Application
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  AppHealthcheckUrl:
+    Type: String
+    Description: The AWS EB health check path
+  AwsAutoScalingMaxSize:
+    Type: String
+    Default: '1'
+  AwsAutoScalingMinSize:
+    Type: String
+    Default: '1'
+  AwsDefaultVpcId:
+    Type: "AWS::EC2::VPC::Id"
+    Description: The AWS account's default VPC id
+  AwsEbHealthReportingSystem:
+    Type: String
+    Default: basic
+    AllowedValues:
+      - basic
+      - enhanced
+    ConstraintDescription: must be either basic or enhanced
+  AwsSnsBounceNotificationEndpoint:
+    Type: String
+    Description: Email address for SNS bounce notifications
+  AwsSnsNotificationEndpoint:
+    Type: String
+    Description: Email address for AWS SNS notifications
+  AwsSolutionStackName:
+    Description: The AWS Solution Stack
+    Type: String
+  BridgeEnv:
+    Type: String
+    Default: dev
+    AllowedValues:
+      - prod
+      - uat
+      - dev
+    ConstraintDescription: must specify dev, uat or prod.
+  BridgeUser:
+    Type: String
+    Default: heroku
+  DNSDomain:
+    Type: String
+    Description: DNS Domain name for deployment
+    Default: sagebridge.org
+  DNSHostname:
+    Type: String
+    Description: DNS Hostname for deployment
+  EC2InstanceType:
+    Type: String
+    Description: Instance type to use for Elastic Beanstalk Instances
+    Default: t2.micro
+    AllowedValues:
+      - t1.micro
```

Comment:
Updated list to t3 hosts, and updated env configs to use t3.smalls and t3.mediums. Also updated this list to remove t1s, t2s, and other host types, since I don't expect we'd ever actually want to launch with anything other than general-purpose hosts.